crates/isograph_lsp/src/goto_definition.rs

```diff
@@ -0,0 +1,181 @@
+use crate::format::char_index_to_position;
+use crate::hover::get_iso_literal_extraction_from_text_position_params;
+use crate::{lsp_runtime_error::LSPRuntimeResult, uri_file_path_ext::UriFilePathExt};
+use common_lang_types::{
+    relative_path_from_absolute_and_working_directory, Location, Span, TextSource,
+};
+use intern::string_key::Lookup;
+use isograph_compiler::CompilerState;
+use isograph_compiler::{get_validated_schema, process_iso_literal_extraction};
+use isograph_lang_types::{DefinitionLocation, IsographResolvedNode};
```

Comment:
I'm not sure why, but your rust analyzer doesn't merge imports :/ anyway nbd I'll merge them haha